packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -149,12 +149,16 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 							{ isFree && __( 'free forever', __i18n_text_domain__ ) }
 							{ ! isFree &&
 								( billingPeriod === 'ANNUALLY'
-									? __( 'billed annually', __i18n_text_domain__ )
-									: __( 'per month, billed monthly', __i18n_text_domain__ ) ) }
+									? sprintf(
+											// translators: %s is the cost per year (e.g "billed as 96$ annually")
+											__( 'billed as %s annually', __i18n_text_domain__ ),
+											planProduct?.annualPrice
+									  )
+									: __( 'billed monthly', __i18n_text_domain__ ) ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Billed monthly' )` *ES Score: 14*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/plans-grid-copy-changes)